modxcms/revolution
# Missing three restricted symbols

*full list symbols are in preg_match function
core/model/modx/processors/security/user/_validation.php:88

### What does it do?
Add missing restricted symbols to lexicon description ";", "<", ">".

### Why is it needed?
You can't using them for passwords.

### How to test
Try to use any of them in new password and you see incomplete list of restricted symbols.

### Related issue(s)/PR(s)
Not such an issue
